modules/gridmap/editor/grid_map_editor_plugin.h

```diff
@@ -187,7 +194,13 @@ class GridMapEditor : public VBoxContainer {
 	Label *info_message = nullptr;
 
 	void update_grid(); // Change which and where the grid is displayed
-	void _draw_grids(const Vector3 &cell_size);
+	void _draw_hex_grid(RID p_grid, const Vector3 &p_cell_size);
+	void _draw_hex_x_axis_grid(RID p_grid, const Vector3 &p_cell_size);
+	void _draw_plane_grid(RID p_grid, const Vector3 &p_axis_n1, const Vector3 &p_axis_n2, const Vector3 &p_cell_size);
+	void _draw_grids(const Vector3 &p_cell_size);
+	void _update_cell_shape(const GridMap::CellShape cell_shape);
```

Comment:
```suggestion
	void _update_cell_shape(GridMap::CellShape cell_shape);
```
No `const` for simple types